### DISCUSSION TITLE
How to turn screen on/off  - Anthias on Pi OS bookworm

### DISCUSSION THREAD

**Discussion by qbicdesign** [Screenly/Anthias#2102] [Show and tell]
To turn the screen off/on  - Anthias on Pi OS bookworm
Firstly:
`sudo nano /boot/firmware/config.txt`
Find the line:
`dtoverlay=vc4-kms-v3d`
change it to:
`dtoverlay=vc4-fkms-v3d`

Now you can set a cron job (for example turn off screens mon-fri 6pm, turn on mon-fri 8am):
```
crontab -e

00 8 * * 1-5 /usr/bin/vcgencmd display_power 1
00 18 * * 1-5 /usr/bin/vcgencmd display_power 0
```

**Comment by nicomiguelino** [comment]
@qbicdesign, changing the value of `dtoverlay` to `vc4-fkms-v3d` will affect video playback, but thanks for sharing!

**Reply by qbicdesign** [reply]
Can you explain a bit further? I just tested it and video playback seems to be unaffected for me on Pi 3 model B.
In what way could it be affected?

**Reply by nicomiguelino** [reply]
Thanks for that info. On some Pi 4 devices, the video playback is not smooth, with noticeable flickering.

**Comment by TobiasKoller** [comment]
if your screen supports CEC (mostly TV's not monitors) then here is a great and easy tutorial: https://pimylifeup.com/raspberrypi-hdmi-cec/

**Reply by nicomiguelino** [reply]
@TobiasKoller, thank you for sharing your thoughs! Anthias utilizes Docker containers, so the results might be different.  In the meantime, feel free to play around with it.

**Reply by TobiasKoller** [reply]
I installed anthias ontop of pi os light and I'm able to control the screen over hdmi now :-)

**Reply by nicomiguelino** [reply]
@TobiasKoller, thank you for confirming this.
